src/instrumentation.ts

```diff
@@ -0,0 +1,12 @@
+export async function register() {
+  if (process.env.NEXT_RUNTIME === 'nodejs') {
```

Comment:
should i add this env variable somewhere

Reply:
also should i add some details wrt testing environment

Reply:
That will not be needed as per my knowledge. 
Will add to Github-Action Env